### PR TITLE
Handle maybe not known responses

### PR DESCRIPTION
**Pull request by raman325**
We have to assume that the response could be undefined, in which case the key wouldn't be in the deserialized dict because typescript does not serialize undefined

### CODE REVIEW

File: zwave_js_server/model/node/__init__.py [hunk 1]
```diff
@@ -689,15 +691,17 @@ async def async_has_security_class(self, security_class: SecurityClass) -> bool:
             wait_for_result=True,
         )
         assert data
```
Comment (MartinHjelmare) [inline zwave_js_server/model/node/__init__.py:693]: This will still raise if `data` is empty.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Comment by MartinHjelmare** [comment]
There are also two keys in the node state that have MaybeNotKnown types in the server. Do we handle those correctly here in the client?

**Comment by raman325** [comment]
> There are also two keys, isSecure and lastSeen, in the node state that have MaybeNotKnown types in the server. Do we handle those correctly here in the client?

fixed `isSecure`, `lastSeen` was already handled correctly

**Review by MartinHjelmare — COMMENTED** [review]
Is the fixture `is_secure_unknown` incorrect? Maybe update it or remove it?

Can we add tests for the new uncovered cases?

**Review by MartinHjelmare — APPROVED** [review]
Change looks good! Just the comments on the tests remaining, which are not critical.

**Comment by raman325** [comment]
> Is the fixture `is_secure_unknown` incorrect? Maybe update it or remove it?
> 
> Can we add tests for the new uncovered cases?

Updated fixture name - for the uncovered paths, I can fake it but I don't have any real dumps that would result in this case. Do you think it's worth manipulating existing state fixtures to achieve the same thing?

**Comment by raman325** [comment]
let me know your thoughts, I will submit a second PR for tests if needed

**Comment by MartinHjelmare** [comment]
I think it would be good to test it. We can adjust the node data before creating the node or similar.

**Pull request merged**